app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="modal fade" id="comment_modal" role="modal">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title text-center"><%= action_name.capitalize %> Comment</h4>
+      </div>
+      <div class="modal-body">
+        <% if action_name == 'new' %>
```

Comment:
why do we have this check here? and what if its not "new"

Reply:
@taimur-akhtar because when the same modal is triggered for the edit comment, we only want to see the comment form fields.